src/compiler/ast/module/SourceFile.ts

```diff
@@ -142,11 +142,15 @@ export class SourceFile extends SourceFileBase<ts.SourceFile> {
     }
 
     /**
-     * Gets the line number at the provided position.
+     * Gets the line number and column number at the provided position.
      * @param pos - Position
      */
-    getLineNumberAtPos(pos: number) {
-        return StringUtils.getLineNumberAtPos(this.getFullText(), pos);
+    getLineAndColumnAtPos(pos: number) {
+        const fullText = this.getFullText();
+        return {
+            line: StringUtils.getLineNumberAtPos(fullText, pos),
+            column: StringUtils.getLengthFromLineStartAtPos(fullText, pos)
```

Comment:
I think we should make column 1 indexed?

Reply:
I don't know, TypeScript getLineAndCharacterOfPosition() is 0 index based for both things. I was surprised noticing 1-indexed for lines in tsMorph. I think we should be as consistent as possible. But since line numbers are already inconsistent with TS and "column" is a different concept than the TS concept  "Character" maybe is a good idea, so both things are consistent between each other. Should I make the change ?

Reply:
@dsherret only waiting confirmation on this and I think that would be all for this PR

Reply:
Yeah, the idea of line and column is usually used by editors and for display purposes so keeping it 1-indexed means that people won't have to convert themselves. Monaco editor is 1-indexed I believe (even in the code).

Let's make it 1-indexed and change the description to `Gets the line and column number at the provided position (1-indexed).` Thanks!

Reply:
on it, you are right, monaco-editor API has the Column and line number concepts and both are  1-indexed